LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -2289,6 +2333,22 @@ private static void GetNuvotonConfigurationD(ISuperIO superIO, Manufacturer manu
                         c.Add(new Ctrl("Chipset Fan", 3));
                         break;
 
+                    case Model.Z790_Taichi:
+                        v.Add(new Voltage("+1.8V PROC Voltage", 0));
```

Comment:
Follow file naming scheme:
PROC > CPU
PCH > Chipset.

Also the + can be removed from 1.8, 0.82 and 1.05V (but keep 3.3V).